portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//EventListener for buttons with dropdowns
+let coll = document.getElementsByClassName("collapse");
+let i;
+
+    for (i = 0; i < coll.length; i++) {
+        coll[i].addEventListener("click", function() {
+            this.classList.toggle("active");
+            let content = this.nextElementSibling;
+            if (content.style.display === "table") {
+                content.style.display = "none";
+            } else {
+                content.style.display = "table";
+            }
+        });
+    }
+
+/**
+ * Grabs the comments from the JSP and displays as a list in HTML 
+ */
+function grabComment(){
+    console.log("Grabbing Comment");
+    fetch("/data").then(response => response.json()).then((list) => {
+        console.log(list);
+        const commentElement = document.getElementById("comment-container");
+        list.forEach((comment) => {
+            commentElement.appendChild(createListElement(comment.name + " said " + comment.content)); 
+        });
+    });
+}
+
+/**
+ * Creates an <li> element containing text.
+ */
+function createListElement(text) {
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    return liElement;
+}   
```

Comment:
This is a great example of a good time to have pulled this out into a function to make the child-appending above very simple

Reply:
Cool beans